godel_simple_gui/src/states/executing_state.cpp

```diff
@@ -64,23 +59,13 @@ void godel_simple_gui::ExecutingState::executeAll(BlendingWidget& gui)
 
 void godel_simple_gui::ExecutingState::executeOne(const std::string& plan, BlendingWidget& gui)
 {
-  /*
-  ROS_DEBUG_STREAM("Executing " << plan);
-  godel_msgs::SelectMotionPlan srv;
-  srv.request.name = plan;
-  srv.request.simulate = false;
-  srv.request.wait_for_execution = true;
-  if (!real_client_.call(srv))
-  {
-    std::ostringstream ss;
-    ss << "Failed to execute plan: " << plan << ". Please see logs for more details.";
-    throw BadExecutionError(ss.str());
-  }
-  */
-  ROS_INFO_STREAM("In select motion plan");
   godel_msgs::SelectMotionPlanActionGoal goal;
   goal.goal.name = plan;
   goal.goal.simulate = false;
   goal.goal.wait_for_execution = true;
-  gui.sendGoal(goal);
+  gui.sendGoalAndWait(goal);
```

Comment:
How does waiting for completion here change the behavior of the gui? would it freeze the rviz panel of completion takes long?